app/assets/javascripts/components/students/components/Articles/SelectedStudent/EditedUnassignedArticles/EditedUnassignedArticles.jsx

```diff
@@ -32,10 +35,18 @@ export const EditedUnassignedArticles = ({
       ...options
     }
   };
+  const getCourseSlug = () => {
```

Comment:
You can get this from `course.slug`.

Reply:
Thanks, I didn't realize. Fixing this now